core/src/com/softwarearchitecture/ECS/Components/TowerComponent.java

```diff
@@ -0,0 +1,27 @@
+package com.softwarearchitecture.ecs.components;
+
+public class TowerComponent {
+    public enum DamageType {
+        FIRE, WATER, LIGHTNING // Sample types
+    }
+
+    public DamageType damageType;
```

Comment:
Could we turn this into an arraylist of DamageType? This lets us add multiple types of damage as we combine two or more cards to create a tower